src/topology/metric_space/thickened_indicator.lean

```diff
@@ -245,3 +246,54 @@ begin
 end
 
 end thickened_indicator -- section
+
+section indicator
+
+variables {α : Type*} [pseudo_emetric_space α]
+
+lemma tendsto_indicator_cthickening_indicator_closure (f : α → ℝ≥0∞) (E : set α) :
```

Comment:
Isn't this lemma true replacing `ℝ≥0∞` by any topological space with a zero? Maybe even give a version with the multiplicative support, like
```lean
variables {α β : Type*} [pseudo_emetric_space α] [topological_space β] [has_one β]

@[to_additive]
lemma tendsto_mul_indicator_cthickening_mul_indicator_closure (f : α → β) (E : set α) :
  tendsto (λ δ, (metric.cthickening δ E).mul_indicator f) 
    (𝓝[>] 0) (𝓝 (mul_indicator (closure E) f)) :=
```
Can't you even state that, eventually, ` (metric.cthickening δ E).mul_indicator f x = mul_indicator (closure E) f x` (without assuming any topology on the target type)?

Reply:
> Isn't this lemma true replacing `ℝ≥0∞` by any topological space with a zero? Maybe even give a version with the multiplicative support, like
> 
> ```lean
> variables {α β : Type*} [pseudo_emetric_space α] [topological_space β] [has_one β]
> 
> @[to_additive]
> lemma tendsto_mul_indicator_cthickening_mul_indicator_closure (f : α → β) (E : set α) :
>   tendsto (λ δ, (metric.cthickening δ E).mul_indicator f) 
>     (𝓝[>] 0) (𝓝 (mul_indicator (closure E) f)) :=
> ```
> 
> Can't you even state that, eventually, ` (metric.cthickening δ E).mul_indicator f x = mul_indicator (closure E) f x` (without assuming any topology on the target type)?

Both are very good ideas, thank you! The general version is surely better, and isolating "eventually for small enough positive delta" lemmas may prove helpful for other purposes. I moved two such "eventually" lemmas to `hausdorff_distance.lean`, because they did not involve indicators.